src/ui/components/shared/Login/Login.tsx

```diff
@@ -177,21 +177,37 @@ function AuthError({ error }: { error: any }) {
     return null;
   }
 
-  if (message === "Invalid state") {
-    // This is usually caused by waiting too long to go through the auth process
-    // and can be fixed by trying again.
-    message = "Your login session expired. Please try logging in again.";
-  } else {
-    // We want to capture any other error so we can investigate further.
+  if (message !== "INVALID_STATE" && message !== "Invalid state") {
     sendTelemetryEvent("devtools-auth-error-login", {
       errorMessage: message,
     });
+  }
 
-    if (message === "Unable to authenticate user") {
+  // Except of "Invalid state" message that comes from
+  // https://auth0.com/docs/customize/integrations/cms/wordpress-plugin/troubleshoot-wordpress-plugin-invalid-state-errors
+  // we define all the other "message"s in our Auth0 Rule.
+  // See more here: https://auth0.com/docs/customize/rules/raise-errors-from-rules
```

Comment:
nit: You might move this comment up, above line 180, where that string is first referenced. Otherwise this is very helpful though 👍🏼